src/rtsp.cpp

```diff
@@ -818,6 +818,12 @@ namespace rtsp_stream {
       ss << "a=rtpmap:98 AV1/90000"sv << std::endl;
     }
 
+    if (!session.surround_params.empty()) {
+      // If we have our own surround parameters, advertise them twice first
+      ss << "a=fmtp:97 surround-params="sv << session.surround_params << std::endl;
+      ss << "a=fmtp:97 surround-params="sv << session.surround_params << std::endl;
```

Comment:
What's the purpose of advertising them twice?

Reply:
The first `surround-params` will be applied to `NormalQualityOpusConfig`, and the second one goes to `HighQualityOpusConfig`. I think advertising them twice first will make sure both configs having the desired mapping.